.github/workflows/element-with-modules.yml

```diff
@@ -0,0 +1,328 @@
+name: Element with modules
+
+on:
+  push:
+    branches: # we want to build the widgets from the push into Element staging and publish a main container
+      - main
+    tags: # we want to build the stable widgets into stable Element and publish the container
+      - '@nordeck/element*module*'
+  pull_request: # we want to build the widgets from the PR into Element develop & staging
+  schedule: # we want to build the stable widgets into stable Element and publish the container, plus a verification of all dev versions
+    - cron: '0 5 * * 1-6'
```

Comment:
Should the days be `1-7` or do we assume nothing is happening on Sundays?

Reply:
actually the intention was `1-5` here: presumably no need to run anything on days where nobody is there on the same day to look at it?

Reply:
:shrug: no strong opinion here